### PR TITLE
Ut 2995 export tangents

### PR DESCRIPTION
**Pull request by BadSingleton**

### CODE REVIEW

File: Source/fbxanimcurve.i [hunk 1]
```diff
@@ -28,6 +28,23 @@
...
+%rename("%s") FbxAnimCurve::KeySetRightDerivative;
+
+%rename("%s", %$isclass) FbxAnimCurveKey;
+%rename("%s") FbxAnimCurveKey::SetTangentMode;
+%rename("%s") FbxAnimCurveKey::SetTangentWeightMode;
+%rename("%s") FbxAnimCurveKey::SetTangentWeightAndAdjustTangent;
+%rename("%s") FbxAnimCurveKey::SetTangentVelocityMode;
+%rename("%s") FbxAnimCurveKey::SetDataFloat;
+%rename("%s") FbxAnimCurveKey::SetTangentVisibility;
+%rename("%s") FbxAnimCurveKey::SetBreak;
```
Comment (benoithudson) [inline Source/fbxanimcurve.i]: feels like we should have getters for these too? Or am I missing something.

Comment (BadSingleton) [reply Source/fbxanimcurve.i]: This was done for an export task I think, so I guess getters aren't MVP?

Comment (BadSingleton) [reply Source/fbxanimcurve.i]: This was done for an export task I think, so I guess getters aren't MVP?

Comment (benoithudson) [reply Source/fbxanimcurve.i]: It's helpful to have getters to debug an exporter.

**Comment by BadSingleton** [comment]
I removed some methods as they are pure virtual and using them don't really have an effect. (as per @vkovec 's advice)

**Pull request merged**